stdlib/_typeshed/__init__.pyi

```diff
@@ -46,6 +46,15 @@ class SupportsDivMod(Protocol[_T_contra, _T_co]):
 class SupportsRDivMod(Protocol[_T_contra, _T_co]):
     def __rdivmod__(self, __other: _T_contra) -> _T_co: ...
 
+# This represents a sequence as defined in the Python documentation:
+# https://docs.python.org/3/glossary.html#term-sequence
+# collections.abc.Sequence and typing.Sequence are not protocols and
+# have additional fields that are often unnecessary.
+class SimpleSequence(Protocol[_T_co]):
+    def __len__(self) -> int: ...
+    def __getitem__(self, __k: int) -> _T_co: ...
+
+# obsolete, use SimpleSequence instead
```

Comment:
Why not just delete it? It was not marked with `# stable`.

Reply:
I guess if it's used in third party stubs (there seems to be the one single case), under some circumstances you could run into silent loss of type checking. This already ends up happening though, e.g. see #5786 and #5751, so I'd also be inclined to just delete it.

If we don't delete it, we should alias here, though, and still update the `annoy` stubs

Reply:
The annoy stubs were my main concern. If we update them, they won't work, until type checkers have been updated with the latest typeshed. I'd suggest keeping it for a few months.